python3/dnf_plugins/ptoken.py

```diff
@@ -3,22 +3,21 @@
 import logging
 import dnf
 
-PTOKEN_PATH = "/etc/xensource/ptoken"
```

Comment:
Keep it as constant, 
and disable the related CI error in test script.

From product point of view, this is a constant
It should NOT be updated and provide setter just for test code.

Reply:
Updated!.
Disabled the pyright warning:
ptoken.PTOKEN_PATH = "/some/not/exist/path" # pyright: ignore[reportAttributeAccessIssue]